DataAccess/Public/Include/DataAccess/QueryBuilder.hpp

```diff
@@ -18,33 +14,33 @@ namespace DataAccess
  * @details The class forms and saves SQL query, current
  * table and type of database. Class provides interface, which is as close as possible
  * to the classical SQL queries. This is achieved by calling methods one by one.
- * Examples for postgre database you can see in PosgreQuery.hpp
+ * Examples for postgre database you can see in PostgreQuery.hpp
  */
 template <typename ResultType>
-class QueryBuilder
+class AbstractQueryBuilder
 {
 private:
-    Utility::DatabaseType       _databaseType;
-    std::string                 _tableName;
-    SQLBase<ResultType>*        _statement;
-
-protected:
-    std::shared_ptr<IAdapter>   _adapter;
+    Utility::DatabaseType     _databaseType;
+    std::string               _tableName;
+    SQLBase<ResultType>*      _statement;
+    std::shared_ptr<IAdapter> _adapter;
 
 public:
-    QueryBuilder(Utility::DatabaseType type, const std::string& tableName, std::shared_ptr<IAdapter> adapter)
-        : _databaseType{ type }, _tableName{ tableName }, _statement{ nullptr }, _adapter{ adapter } {}
+    AbstractQueryBuilder(Utility::DatabaseType type, const std::string& tableName, std::shared_ptr<IAdapter> adapter)
+        : _databaseType{type}, _tableName{tableName}, _statement{nullptr}, _adapter{adapter}
+    {
+    }
 
-    virtual ~QueryBuilder(void) { this->clearStatement(); }
+    virtual ~AbstractQueryBuilder(void) { this->clearStatement(); }
```

Comment:
In C++, the default in parentheses when no arguments are given is void. Please fix this